conf/generators/assignment.json

```diff
@@ -0,0 +1,34 @@
+{
+	"name": "assignment_op",
+	"fname": "11.4.1--{1}_assignment.sv",
+	"templ": [
```

Comment:
Since it is becoming a kind of API for tests generators, maybe we could use the full `template` word here?

Reply:
Good idea, should `fname` be expanded to `file_name` too?